roles/preflight-checks/tasks/subnet.yml

```diff
@@ -0,0 +1,63 @@
+---
+- name: checking neutron subnet {{ item.name }}
+  os_subnets_facts:
+    name: "{{ item.name }}"
+    auth:
+      auth_url: "{{ endpoints.auth_uri }}"
+      project_name: admin
+      username: admin
+      password: "{{ secrets.admin_password }}"
```

Comment:
Will this catch subnets that the customer creates themselves? Or will it limit itself to just those that exist in the admin account?